administrator/components/com_users/src/Controller/UserController.php

```diff
@@ -160,4 +161,34 @@ public function batch($model = null)
     protected function postSaveHook(BaseDatabaseModel $model, $validData = [])
     {
     }
+
+    /**
+     * Get the XHR request to activate a user
+     * js: media/com_users/js/activate-user-send-email
+     *
+     * @return void
+     *
+     * @since  __DEPLOY_VERSION__
+     */
+    public function active(): void
+    {
+        // Get the ID of the user
+        $userId          = $this->input->getString('userid', '');
```

Comment:
```suggestion
        $userId = $this->input->getString('userid', '');
```
`=`should only be aligned in consecutive lines, but here this line stands alone. No idea why the CS checker did not complain.